starter_code/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>CSS Practice</title>
+  </head>
+  <body>
+    <header>Relaxr</header>
+    <h2>Get piece of mind with a single tap</h2>
+
+    <img src="https://github.com/generalassembly-atx/relaxr_html/blob/master/starter_code/images/header_bg.jpg?raw=true" alt="ADA" />
+
+    <button type="button" name="get-now">Get it Now</button>
+
+    <div class="benefits">
+    <main>
+      <h4>Benefits</h4>
+      <br>
+      The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.
+      <br>
+      <ul>
+        <li>Schedule meetings for you</li>
+        <li>Excel automation to complete your work for you</li>
+        <li>Responds to emails on your behalf</li>
+        <li>Does all of your work for you with our revolutionary AI technology</li>
+      </ul>
+    </main></div>
```

Comment:
Wrapping a `<div>` around `<main>` might be superfluous. I'd go with one or the other (with the class of "benefits").